lib/src/command/get.dart

```diff
@@ -60,6 +67,50 @@ class GetCommand extends PubCommand {
     );
   }
 
+  bool shouldRunPostGetHook() {
+    final flutterRoot = Platform.environment['FLUTTER_ROOT'];
+    if (flutterRoot == null) {
+      return false;
+    }
+    if (!fileExists(p.join(flutterRoot, 'version'))) {
+      return false;
+    }
+
+    // TODO: sky_engine check
+    final hasFlutterDependency =
+        entrypoint.root.dependencies.values.any((package) {
+      return package.name == 'flutter' &&
+          package.source.runtimeType == SdkSource;
+    });
+
+    return hasFlutterDependency;
+  }
+
+  Future<int> runPostGetHook() async {
+    final String flutterRoot = Platform.environment['FLUTTER_ROOT']!;
+    final String flutterToolPath = p.join(flutterRoot, 'bin', 'flutter');
+
+    final StreamSubscription<ProcessSignal> subscription =
+        ProcessSignal.sigint.watch().listen((e) {});
```

Comment:
Why are we doing this? We should at least have a comment explaining this.

Reply:
Now that I think about it, I'm not sure why I had that either 😄.